pretext/SimplePythonData/ValuesandDataTypes.ptx

```diff
@@ -26,21 +26,14 @@ print("Hello, World")
     <p>When we show the value of a string using the <c>print</c> function, such as in the third example above, the quotes are not present in the output.  The
                 value of the string is the sequence of characters inside the quotes.  The quotes are only necessary to help Python know what the value is.</p>
   </note>
-  <p>You may have used function notation in a math class, like <c>y = f(x)</c>, likely only for functions that act on a single numeric value, and produce a single numeric value.  Python has no such restrictions:  Inputs and outputs may be of any type.</p>
-  <p>In the Python shell, it is not necessary to use the <c>print</c> function to see the values shown above.  The shell evaluates the Python function and automatically prints the result.  For example, consider the shell session shown below.  When
-            we ask the shell to evaluate <c>type("Hello, World!")</c>, it responds with the appropriate answer and then goes on to
-            display the prompt for the next use.</p>
-  <pre>Python 3.1.2 (r312:79360M, Mar 24 2010, 01:33:18)
-[GCC 4.0.1 (Apple Inc. build 5493)] on darwin
-Type "help", "copyright", "credits" or "license" for more information.
-&gt;&gt;&gt; type("Hello, World!")
-&lt;class 'str'&gt;
-&gt;&gt;&gt; type(17)
-&lt;class 'int'&gt;
-&gt;&gt;&gt; "Hello, World"
-'Hello, World'
-&gt;&gt;&gt;</pre>
-  <p>Note that in the last example, we simply ask the shell to evaluate the string <q>Hello, World</q>.  The result is as you might expect, the string itself.</p>
+  <p>You may have used function notation in a math class, like <c>y = f(x)</c>, likely only for functions that act on a single numeric value, and produce a single numeric value.  
+  In Python we have no such restrictions: In programming, a function is like a mini-program designed to perform a specific task. When you "call" a function, you're asking it to execute its task. 
```

Comment:
I'd be careful about calling a function a "mini-program" .. I might call it like a subroutine, or task (as you call it a task anyways later)

Also, the function notation analogy might be confusing, we might just want to say directly what a function is and what it looks like

Reply:
removed analogy, change mini-program